core/config/locales/en.yml

```diff
@@ -595,6 +595,7 @@ en:
       spree/store:
         one: Store
         other: Stores
+        default: Default
```

Comment:
This needs to be in the `activerecord.attributes` section

Reply:
fixed